cobalt/h5vcc/h5vcc_crash_log.cc

```diff
@@ -174,5 +174,29 @@ void H5vccCrashLog::SetPersistentSettingWatchdogCrash(bool can_trigger_crash) {
   if (watchdog) watchdog->SetPersistentSettingWatchdogCrash(can_trigger_crash);
 }
 
+void H5vccCrashLog::LogEvent(std::string event) {
+  watchdog::Watchdog* watchdog = watchdog::Watchdog::GetInstance();
+  if (watchdog) watchdog->LogEvent(event);
+}
+
+script::Sequence<std::string> H5vccCrashLog::GetLogTrace() {
+  watchdog::Watchdog* watchdog = watchdog::Watchdog::GetInstance();
+
+  script::Sequence<std::string> sequence;
+  if (watchdog) {
+    std::vector<std::string> logTrace = watchdog->GetLogTrace();
```

Comment:
Interesting, why the need to convert to vector here? You already have a Sequence. I see other examples like this elsewhere, but not sure why. Do you?

Reply:
I guess you mean the other way around: from vector to sequence. 
I don't understand the root cause either, curious as well. But there's no constructor or any other way to copy https://github.com/youtube/cobalt/blob/bbbef4cc0cc600592c234edaa45331ec2a96d6e0/cobalt/script/sequence.h#L28
I suppose it has something to do with WebIDL JS to cpp integration.